app/assets/data/bebbo/sponsers.ts

```diff
@@ -0,0 +1,361 @@
+
+import { nmSponserLogo } from "@dynamicImportsClass/dynamicImports";
+
+const sponsersall = [
+  {
+    id: 6,
+    type: 'Country',
+    title: "Albania",
+    country_flag: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_sponsor_logo: {
+      url: "",
+      name: "",
+      alt: ""
+    },
+    country_national_partner: {
+      name: "IPH.png",
+      alt: "IPH"
+    },
+    make_available_for_mobile: 0,

```

Comment:
make_available_for_mobile property can be removed if it is always 0. Otherwise assign a Boolean value to make it meaningful.